dashboard/pkg/epinio/edit/services.vue

```diff
@@ -134,6 +134,13 @@ export default Vue.extend<Data, EpinioCompRecord, EpinioCompRecord, EpinioCompRe
   methods: {
     async save(saveCb: (success: boolean) => void) {
       this.errors = [];
+
+      const newSettings = !isEqual(objValuesToString(this.chartValues), objValuesToString(this.value.settings));
```

Comment:
`objValuesToString(this.chartValues)` is called twice. This can be done just once above `const chartValues = objValuesToString(this.chartValues)`